edx_sga/sga.py

```diff
@@ -554,16 +554,45 @@ def get_staff_grading_data(self, request, suffix=''):
         require(self.is_course_staff())
         return Response(json_body=self.staff_grading_data())
 
+    def validat_score_message(self, course_id, username):
```

Comment:
I didn't see this earlier, but it should be `validate_score_message` 

You dropped an `e` at the end of `validate`